src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -41,12 +41,12 @@ public static void CompareEntries(string baselineFileName, IOrderedEnumerable<st
             Assert.Null(message);
         }
 
-        public static void CompareBaselineContents(string baselineFileName, string actualContents, ITestOutputHelper outputHelper, bool warnOnDiffs = false)
+        public static void CompareBaselineContents(string baselineFileName, string actualContents, ITestOutputHelper outputHelper, bool warnOnDiffs = false, bool isLicenseBaseline = false)
```

Comment:
Would it make sense to abstract this instead of hardcoding the license concept?  For example, an optional subfolder parameter that would allow the licenses tests to control as well as other future tests.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/959fa223c3bcdc496b635db83a329eaa9aad5f98